render/topologies.go

```diff
@@ -116,6 +118,56 @@ var ContainerRenderer = MakeReduce(
 	),
 )
 
+type containerWithHostIPsRenderer struct {
+	Renderer
+}
+
+// Render produces a process graph where the ips for host network mode are set
+// to the host's IPs.
+func (r containerWithHostIPsRenderer) Render(rpt report.Report) RenderableNodes {
+	containers := r.Renderer.Render(rpt)
+	hosts := MakeMap(
+		MapHostIdentity,
+		SelectHost,
+	).Render(rpt)
+
+	for id, c := range containers {
+		networkMode, ok := c.Node.Latest.Lookup(docker.ContainerNetworkMode)
+		if !ok || networkMode != docker.NetworkModeHost {
```

Comment:
How about extending this to show the IPs of the parent container when `-net=containerID`?